pytd/writer.py

```diff
@@ -33,7 +35,7 @@ def _cast_dtypes(dataframe, inplace=True):
 
     for column, kind in dataframe.dtypes.apply(lambda dtype: dtype.kind).iteritems():
         if kind == "i" or kind == "u":
-            t = int
+            t = float if df[column].isnull().any() else int
```

Comment:
Just FYI, might be better

```suggestion
            t = pd.Int64Dtype() if df[column].isnull().any() else int
```

Reply:
Thank you for the suggestion. I think you missed my comments and commits above...

Reply:
yeah, I overlooked your latest modification. will review again